util/localconfig/localconfig.go

```diff
@@ -303,3 +313,11 @@ func GetUsername(subject string) string {
 	}
 	return subject
 }
+
+func GetFilePermission(fi os.FileInfo) error {
+	//argo directory should have mode 0700 & file should have permission -  0600.
+	if fi.Mode().Perm() == 0600 {
+		return nil
+	}
+	return fmt.Errorf("config file has incorrect permission flags: %s  change the permission to 0600(linux permission: -rw-------)", fi.Mode().Perm().String())
```

Comment:
Would you be opposed to inlining this? It's fairly self-explanatory and could save a few lines of code.

Reply:
That's fine!